dev-util/rpi-imager/rpi-imager-1.8.5.ebuild

```
@@ -0,0 +1,66 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
```

```suggestion
inherit cmake xdg
```
Inherit xdg eclass to update icon/desktop file caches.